backend/tests/unit/core/schema_manager/test_validate_schema_path.py

```diff
@@ -13,7 +13,7 @@
 
 class TestValidateSchemaPath:
     @pytest.fixture
-    async def schema(db: InfrahubDatabase, reset_registry, default_branch: Branch) -> SchemaBranch:
+    async def schema(self, db: InfrahubDatabase, reset_registry, default_branch: Branch) -> SchemaBranch:
```

Comment:
This looks a bit weird that it worked previously but would be related to the fact that we didn't actually use the `db` variable in the function.